tools/regression_test.py

```diff
@@ -519,6 +520,52 @@ def get_info_from_log_file(info_type: str, log_path: Path,
     return info_value
 
 
+def run_cmd(cmd_lines: List[str], log_path: Path):
+    """
+    Args:
+        cmd_lines: (list[str]): A command in multiple line style.
+        log_path (Path): Path to log file.
+
+    Returns:
+        int: error code.
+    """
+    return_code = 1
+    import platform
+    system = platform.system().lower()
+
+    if system == 'windows':
+        sep = r'`'
+    else:  # 'Linux', 'Darwin'
+        sep = '\\'
+    cmd_str = f' {sep}\n'.join(cmd_lines) + '\n'
```

Comment:
use` cmd_str = f' '.join(cmd_lines) on windows`

Reply:
> use` cmd_str = f' '.join(cmd_lines) on windows`
@irexyc 
It's for better human visualization of multiple lines. Is the separator ` OK for windows?